cluster-provision/gocli/pkg/libssh/ssh.go

```diff
@@ -224,9 +227,24 @@ func (s *SSHClientImpl) executeCommand(cmd string, outWriter, errWriter io.Write
 func (s *SSHClientImpl) initClient() error {
 	s.initMutex.Lock()
 	defer s.initMutex.Unlock()
-	client, err := ssh.Dial("tcp", net.JoinHostPort("127.0.0.1", fmt.Sprint(s.sshPort)), s.config)
+
+	var (
+		client *ssh.Client
+		err    error
+	)
+
+	operation := func() error {
+		client, err = ssh.Dial("tcp", net.JoinHostPort("127.0.0.1", fmt.Sprint(s.sshPort)), s.config)
+		return err
```

Comment:
Why did you remove the more verbose error message here?

Reply:
That's fine, I can reintroduce it. I probably just removed it unintentionally